lib/helpers/tasks.rb

```diff
@@ -65,59 +65,46 @@ def generate_types(environment)
     notify_slack(status_message) if slack?
```

Comment:
This (and line 61) should be `notification()`

Reply:
Nice catch! I'll take care of that.